main.css

```diff
@@ -114,3 +115,328 @@ ul.social {
 li.social-icon {
   display: flex;
 }
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  position: static;
+  width: 390px;
+  height: 600px;
+  left: calc(50% - 327px / 2);
+  top: 114px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 88px 0;
+}
+
```

Comment:
[ Required Changes ]
You're doing great bro, 👍.

- [ ] Please add margin-left and margin-right for each project so your design will match the Figma design.
Your design
![image](https://user-images.githubusercontent.com/50941074/158983307-1dcbb91f-de1e-4f2f-a3f6-0dc8d5016d2e.png)

The Figma Design
![image](https://user-images.githubusercontent.com/50941074/158983142-fa15db92-6005-468e-9441-416211898bf4.png)

Please fix that.
Hint: you can replace your line `136` with `margin: 88px 20px;`  🤝